internal/testtypes/builders.go

```diff
@@ -259,7 +293,7 @@ func NewInfrastructureMachineTemplateBuilder(namespace, name string) *Infrastruc
 	}
 }
 
-// WithSpecFields will add fields of any type to the object spec. It takes an argument, fields, which is of the form path: object.
+// WithSpecFields will add matchers of any type to the object spec. It takes an argument, matchers, which is of the form path: object.
```

Comment:
Is this intentional? (same for most of the s/Fields/Matchers)

Reply:
Needs to be updated.